pkg/collect/host_network.go

```diff
@@ -63,7 +54,8 @@ func isValidLoadBalancerAddress(address string) bool {
 	}
 
 	// Checking if it's all numbers and .
-	if isValidIPCandidate(hostAddress) {
+	var ipRegexp = regexp.MustCompile(`^[0-9.]+$`)
```

Comment:
This should be at the top of the file just after the constants so it fails immediately. If it's in here you'll have to use `Compile` and check the returned error is nil.